src/test/java/com/google/cloud/teleport/splunk/CustomX509TrustManagerTest.java

```diff
@@ -47,13 +47,11 @@ public void setUp()
         new FileInputStream(classLoader.getResource("PubsubToSplunkTestData/RootCA.crt").getFile());
     FileInputStream recognizedInputStream =
         new FileInputStream(
-            classLoader
-                .getResource("PubsubToSplunkTestData/RootCASignedCertificate.crt")
-                .getFile());
+            classLoader.getResource("PubsubToSplunkTestData/RecognizedCertificate.crt").getFile());
     FileInputStream unrecognizedInputStream =
         new FileInputStream(
             classLoader
-                .getResource("PubsubToSplunkTestData/UnrecognizedSelfSignedCertificate.crt")
```

Comment:
again if you compare the latest versions of the branches they both look like the right, so github is showing a non-existent change here